### DISCUSSION TITLE
Disable reply notifications

### DISCUSSION THREAD

**Discussion by hundredfire** [Docile-Alligator/Infinity-For-Reddit#1621] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

We used to have the option to disable notifications of replies to comments right from the discussion thread. I can't find any mention of why this was removed. Would it be possible to have this feature back?

### Alternatives

If this is not possible on a per-comment basis, perhaps adding a global option would be interesting?

### Additional context

I used this feature very often for comments that don't need answers for, or for which you don't want to expand on. Can also be good if you are flooded by replies from a single comment.

### How important is this feature?

Nice to have

**Comment by hundredfire** [comment]
No one interested? 😔

**Comment by mrbigmouth502** [comment]
I've been wanting this feature for years. Upvoted.

**Comment by Docile-Alligator** [comment]
Done. Will be available in the next update. https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/56379101e1e9fd05a1ec3c652ecb9d208dd4b653
